### ISSUE TITLE
Clipping paths

### ISSUE THREAD

**Issue by projectitis** [blend2d/blend2d#141]
I would like to write a Rive renderer using Blend2D. It looks like BL supports all the required features, except I can't find reference to clipping paths (only clipRect) in the docs.

Are clip paths planned for the future?

This would be the only blocker to a Rive renderer at first glance.

**Comment by kobalicek** [comment]
Yeah clipping paths are planned in the future, and multiple layers too.

We should do something about this after geometry changes.

**Comment by na2axl** [comment]
I'm facing the exact same issue for the exact same purpose. Wanted to implement a Rive renderer for a game engine, but I'm missing only the clip path feature to complete it...

**Comment by Jipok** [comment]
Any progress?
